lib/client.js

```diff
@@ -591,12 +591,24 @@ extend(Raven.prototype, {
       'url'
     ];
 
-    nonEnumberables.forEach(function(key) {
+    nonEnumerables.forEach(function(key) {
       sources.forEach(function(source) {
         if (source[key]) request[key] = source[key];
       });
     });
 
+    /**
+     * Check for 'host' *only* after we checked for 'hostname' first.
+     * This way we can avoid the noise coming from Express deprecation warning
+     * https://github.com/expressjs/express/blob/b97faff6e2aa4d34d79485fe4331cb0eec13ad57/lib/request.js#L450-L452
+     * REF: https://github.com/getsentry/raven-node/issues/96#issuecomment-354748884
+     **/
+    if (!request.hasOwnProperty('hostname')) {
+      sources.forEach(function(source) {
```

Comment:
Iteration is pointless

Reply:
It's not because we want to keep correct order or eventual overrides – https://github.com/getsentry/raven-node/blob/master/lib/client.js#L229-L247